packages/harpguru-core/src/components/harp-guru/utils/get-initial-global-state/get-initial-global-state.ts

```diff
@@ -62,6 +62,7 @@ export const getInitialGlobalState = (pageNumber: PageNumber): GlobalState => {
     activeDisplayMode: initialDisplayMode,
     lockedCovariant: initialLockedCovariant,
     bufferedActivityToggles: thisPozitionDegrees,
+    isOverridden: false,
```

Comment:
This name can be improved to be more specific.

The type of the property should be changed to an enumerator too so that it's clear that there can be any number of flush channels; they just need to request and relinquish control from and to the normal flusher.

Reply:
Done